sources/packages/web/src/views/student/NoticeOfAssessment.vue

```diff
@@ -9,6 +9,7 @@
           params: {
             applicationId: applicationId,
             assessmentId: assessmentId,
+            currentAssessmentId: currentAssessmentId,
```

Comment:
Same from https://github.com/bcgov/SIMS/pull/3332/files#r1614087252
Please check if it makes sense.

Reply:
Updated